visualization/app/codeCharta/ui/copyToClipboardButton/util/clipboardString.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { buildTextOfFiles } from "./clipboardString"
+import { FileToValue } from "./getFilenamesWithHighestMetrics"
+
+describe("buildTextOfFiles", () => {
+	it("should return only header line if there is only one attribute without files", () => {
+		const result = buildTextOfFiles(WITH_ONE_FILELESS_ATTRIBUTE)
+
+		expect(result).toBe("FUNCTIONS\n")
```

Comment:
is this useful, or should we skip a header without entries?

Reply:
is this possible at all, or is this attribute then not included in cc.json anyway? I am not sure. Anyway, it is an not important edge case, so let's not spend too much time with it.

Reply:
It is good to know that the function can handle this edge case without throwing an exception (fail safe).